local-config/config.yaml

```diff
@@ -0,0 +1,10 @@
+http:
+  prefix: someprefix
```

Comment:
this looks like it shouldn't be here

Reply:
We are not even using this file so I just deleted it 😅